### ISSUE TITLE
How to get back input and predictions as string

### ISSUE THREAD

**Issue by bikashg** [huggingface/pytorch-pretrained-BERT#553]
Once I am done fine tuning my `BertForSequenceClassification` model, I evaluate it on a validation set. I can see the loss and accuracy scores but I would also like to get the actual labels (as string) it predicted for each sentence (string) in the validation dataset. How could I do that?
